packages/sources/coinmetrics/src/endpoint/burned.ts

```diff
@@ -1,63 +1,164 @@
-import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
 import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
-import { NAME as AdapterName } from '../config'
 import { ethers } from 'ethers'
+import { Decimal } from 'decimal.js'
+Decimal.set({ precision: 100 })
```

Comment:
Let's not introduce decimal.js unless we have to. Ethers' BigNumber library is our preferred BN library, and it can parse out the amounts we're dealing with here.